### ISSUE TITLE
Test Results output is empty

### ISSUE THREAD

**Issue by Godrules500** [jest-community/vscode-jest#1094]
### Environment

1. `vscode-jest version`: Jest Orta 5.1.0
2. `node -v`: 18.15.0
3. `npm -v` or `yarn --version`: 9.8.1
4. `npm ls jest` or `npm ls react-scripts` (if you haven’t ejected): npm is jest
5. your vscode-jest settings if customized:
    - jest.jestCommandLine? [fill]
    - jest.autoRun? [fill]
    - anything else that you think might be relevant? [fill]
6. Operating system: windows

### Prerequisite
- are you able to run jest test from the command line? yes
- how do you run your tests from the command line? (for example: `npm run test` or `node_modules/.bin/jest`) npm run test

### Steps to Reproduce

Expected behavior -
In VS code's test explorer window, I click the run button
After all tests run, I get a summary results. I also would be able to click on a failed test and see why it failed.

Actual behvaior -
All tests run, but I get no output console, nor do I get a summary. If I click on the individually failed test, I get nothing.

I am running in typescript, not sure if that matters.

![Vscode test results output](https://github.com/jest-community/vscode-jest/assets/318063/5b007de0-8c39-42b0-955d-4cb76914b697)

**Comment by mirceaburac-sugarcrm** [comment]
Having the same problem.

I've tried playing around with these settings, but nothing worked:
```
"jest.runAllTestsFirst": false,
  "jest.autoClearTerminal": false,
  "testing.openTesting": "openOnTestStart",
  "jest.outputConfig": {
    "revealOn": "run",
    "revealWithFocus": "terminal",
    "clearOnRun": "none"
  },
  "jest.runMode": "on-demand",
  "jest.autoRun": {
    "watch": false
  }
```

**Comment by connectdotz** [comment]
This should be fixed in [vscode-jest v6.1.0 pre-release](https://github.com/jest-community/vscode-jest/blob/master/release-notes/release-note-v6.md#v610-pre-release). Please give the pre-release a try and let me know if it resolves your problem.

**Comment by mirceaburac-sugarcrm** [comment]
@connectdotz That fixed it for me, thank you.
